pytorch_lightning/loggers/comet.py

```diff
@@ -49,6 +49,14 @@
     CometExperiment, CometExistingExperiment, CometOfflineExperiment = None, None, None
     API = None
 
+_MATPLOTLIB_AVAILABLE = _module_available("matplotlib")
+
+if _MATPLOTLIB_AVAILABLE:
+    import matplotlib.pyplot as plt
+else:
+    from pytorch_lightning.utilities.mock_types import matplotlib as _matplotlib
+    plt = _matplotlib.pyplot
+
```

Comment:
Is it worth adding this just for the type annotation?

Reply:
Added this because someone suggested it above. Does not hurt, does it?

Reply:
--> Needs review decision